packages/botonic-plugin-nlu/src/model-handler.ts

```diff
@@ -42,23 +45,19 @@ export class ModelHandler {
     return preprocessor
   }
 
-  async loadModelInformation(options: BotonicPluginNLUOptions): Promise<this> {
+  async loadModelInformation(
+    options: BotonicPluginNLUOptions
+  ): Promise<ModelHandler> {
+    const promises: { [lang: string]: ModelInformationPromises } = {}
```

Comment:
use a map to avoid having to type.
const promises = this.languages.map(l => getModelInfoFromEnv(lang))